composer.json

```diff
@@ -23,12 +23,12 @@
         }
     ],
     "require": {
-        "php" : "^7.2",
-        "illuminate/support": "^7.0|^6.0|^5.5"
+        "php" : "^7.3",
+        "illuminate/support": "^8.0"
```

Comment:
Hi @khatabwedaa, can you please keep the old versions and add `^8.0` as an additional version? 
Cause this will break old versions support.